src/spac/transformations.py

```diff
@@ -320,77 +325,47 @@ def rename_labels(adata, src_annotation, dest_annotation, mappings):
     mappings : dict
         A dictionary mapping the original annotation labels to
         the new labels.
-
-    Returns
-    -------
-    adata : anndata.AnnData
-        The updated Anndata object with the new column containing the
-        renamed labels.
-
-    Examples
-    --------
-    >>> adata = your_anndata_object
-    >>> src_annotation = "phenograph"
-    >>> mappings = {
-    ...     "0": "group_8",
-    ...     "1": "group_2",
-    ...     "2": "group_6",
-    ...     # ...
-    ...     "37": "group_5",
-    ... }
-    >>> dest_annotation = "renamed_annotations"
-    >>> adata = rename_annotations(
-    ...     adata, src_annotation, dest_annotation, mappings)
+    layer : str, optional
+        The name of the layer in the AnnData object to check.
+    feature : str, optional

```

Comment:
@fangliu117, I don't see the feature argument used in this function.

Reply:
Yes, you are correct that the feature parameter is not necessary, so I have removed it.